_docs_v7/Slope-Limiters-and-Shock-Resolution.md

```diff
@@ -0,0 +1,193 @@
+
+Before mathematically describing the form of the limiters implemented in SU2, it is useful to briefly understand two concepts. These include **Total Variation** and **Godunov's Theorem**. 
+
+### Total Variation and Total Variation Diminishing
+We can first introduce the concept of **total variation** (TV) which is a measure of how oscillatory a solution is. In a discrete one dimensional setting, TV can be calculated as the following:  
+
+$$ TV(u^n) = \sum_j |u^n_{j+1} - u^n_j| $$
+
+<img src="../../docs_files/TV_example.png" width="500">
+
+Figure (2): A numerical scheme resulting in both high and low TV.  
+
+A scheme can be said to be **total variation diminishing** (TVD) if 
+
+$$ TV(u^{n+1}) \leq TV(u^n) $$
+
+where for every successive timestep $$n$$, the total variation of the solution does not increase. 
+
+A favorable property of TVD schemes is that they are **monotonicity preserving**. This means they do not introduce new extrema into the solution and local minimum (maximum) are non-decreasing (increasing) in time. These are both desirable qualities for correctly resolving a shock and ensuring the solution is physical. 
+
+### Godunov's Theorem
+The question of "How accurate can a TVD scheme be?" is still unanswered. For this, we turn to [Godunov's Theorem](https://en.wikipedia.org/wiki/Godunov%27s_theorem).
+
+**Godunov's Theorem**: 
+1. A linear scheme is monotone if and only if it is total variation diminishing. 
+2. Linear total variation diminishing schemes are at most first-order accurate. 
+
+The first statement is simple, stating that for linear schemes, the characteristic of being monotone and TVD is equivalent. The second statement is more interesting. It states that if we want to construct a linear TVD (monotone) scheme, the best we can be possibly hope for is first-order accuracy. 
+
+Recall that the original motivation for a slope limiter was to prevent the formation of oscillations in the solution. In the section above, we noted that TVD schemes are monotonicity preserving (a favorable property in resolving a shock). However, through Godunov's theorem, we note that if we also want high-order accuracy, **our TVD discretization MUST be nonlinear**
+
+The inclusion of a slope limiter into a TVD scheme accomplishes this idea. 
+
+
+
+<!-- ??? do we need to include def of venkat function??? -->
+<!-- It would make more sense to include the limiters in the section above -->
+
+
+## Available Limiter Options
+
+The field `SLOPE_LIMITER_FLOW` in the `.cfg` file specifies which limiter to use. Note that this option is only used if `MUSCL_FLOW = YES` (which specifies to use a second-order method).
+The [Laminar Cylinder](https://su2code.github.io/tutorials/Laminar_Cylinder/) shows an example of this.
+The [Turbulent Flat Plate example](https://su2code.github.io/tutorials/Turbulent_Flat_Plate/) sets `SLOPE_LIMITER_TURB`, which is used for the turbulence equations, rather than for the flow equations.
```

Comment:
```suggestion
The [Turbulent Flat Plate example](https://su2code.github.io/tutorials/Turbulent_Flat_Plate/) sets `SLOPE_LIMITER_TURB`, which is used for the turbulence equations (if `MUSCL_TURB = YES`), rather than for the flow equations.
```